.circleci/config.yml

```diff
@@ -6,8 +6,8 @@ version: 2
 jobs:
   build:
     docker:
-      # This Cloud Function uses the Node.js 8 runtime
-      - image: circleci/node:8.15
+      # This Cloud Function uses the Node.js 10 runtime
+      - image: circleci/node:10.18.1
```

Comment:
10.18.1 is specified by Google: https://cloud.google.com/functions/docs/concepts/nodejs-runtime#execution_environment